### ISSUE TITLE
world crashes relating to thaumcraft

### ISSUE THREAD

**Issue by Gamejet** [pyure/infitech-prototype#267]
In TC 6.1 Beta 26 theres a known issue with Arcane Pedestals where they bug out. They get unuseable and even crash the server  (issue: https://github.com/Azanor/thaumcraft-beta/issues/1505). This got fixed on oct 18th, Unfortinatly the hotfix (has the same filename as the original file: https://minecraft.curseforge.com/projects/thaumcraft/files/2629023) hasnt made its way into the pack yet.

**Comment by Agrax** [comment]
It is in the pack. We have TC beta 26 ver 2

**Comment by Gamejet** [comment]
Well thats weird. We ran into precisely that issue and an update to the fixed TC version fixed it.

**Comment by Agrax** [comment]
Well... thanks for the issue then. Weird, but okay. We’ll look into it

**Comment by Gamejet** [comment]
When we updated our server from 0.3.2 to 0.3.4 it would crash every time someone logged in and loaded an arcane pedestal with that. The problem was the server tried to load the fixed Arcane Pedestals in the world with the non-hotfixed version of TC (When I upgraded I replaced all mod files with the ones from 0.3.4). I had to reupdate TC again to fix it; long story short: The thaumcraft version in the pack is still not fixed.
